### DISCUSSION TITLE
Concurrent conversion issue while using libreoffice.

### DISCUSSION THREAD

**Discussion by sauravsinha137** [gotenberg/gotenberg#893] [Q&A] [answered]
My expectation is I want to process multiple docx to pdf conversion parallel
I am doing a test on this service to validate if we can use this. 

This test was performed using a K6 script on a service running locally in a docker container with configuration as
2 core cpu.
2 gb memory.

1. Run for 60s with 1 Virtual user. (screenshot 1)
No. of requests served: 24
http_req_duration..............: avg=2.59s    min=2.54s med=2.58s max=2.72s  p(90)=2.62s    p(95)=2.63s
2. Run for 60s with 2 Virtual user. (screenshot 2)
No. of requests served: 25
 http_req_duration..............: avg=5.1s     min=2.72s med=5.16s max=5.36s  p(90)=5.31s   p(95)=5.33s
3. Run for 60s with 3 Virtual user. (screenshot 3)
No. of requests served: 25
http_req_duration..............: avg=7.6s     min=2.74s med=7.92s max=8.19s  p(90)=8.03s   p(95)=8.08s

Latency increases with increase in parallel execution. I wish to avoid this. Before 8 you had this --unoconv-disable-listener
I believe this can be used to avoid single running instance  of libreoffice instead create a separate instance for each request. I am hoping parallel execution can be improved with this. But i see this flag is Deprecated in 8.
How can i solve this problem with gotenberg 8.

**Comment by gulien** [comment, marked as answer]
Hello @sauravsinha137,

That's because now you'll have to handle it in a Docker-way, i.e., auto-scale or set up a determined number of Gotenberg instances. Most if not all Docker orchestrators offer features for container scaling.

For instance, with Docker Compose:

```yaml
services:
  # Your other services.

  gotenberg:
    image: gotenberg/gotenberg:8
    ports:
      - "3000:3000"
```

```bash
$ docker-compose scale gotenberg=5
```

**Reply by sauravsinha137** [reply]
Thanks @gulien for quickly reply. I guess creating multiple instances are the only way improve this.
